Dockerfile.ubuntu

```diff
@@ -60,6 +49,3 @@ RUN pip install -r /opt/file-metadata/test-requirements.txt
 
 # Install package
 RUN pip install /opt/file-metadata --upgrade
-
-# Test file-metadata import
-RUN python -c'import file_metadata; print(file_metadata.__version__)'
```

Comment:
We need some command to run in the end to test the successful install. At least an import test.